PWGCF/FEMTOSCOPY/FemtoDream/AliAnalysisTaskPOmegaPenne.cxx

```diff
@@ -1074,17 +833,17 @@ void AliAnalysisTaskPOmegaPenne::UserCreateOutputObjects()
     
     tlLambdaList            = fLambdaV0Cuts->GetQAHists();
     tlAntiLambdaList        = fAntiLambdaV0Cuts->GetQAHists();
-    tlCascadeCutsXi         = fCascadeCutsXi->GetQAHists();
-    tlAntiCascadeCutsXi     = fCascadeCutsAntiXi->GetQAHists();
 
 // initialize and connect RESULTS
-    // RESULTS 1
+    // RESULTS 1 /////////////////////////////////////////////
     if (fConfig->GetUseEventMixing())
     {
         tlResults = fPartColl->GetHistList();
```

Comment:
I am not 100% sure about naming conflicts and how well this works. So What I would do it after you do GetHistList() you call tlResults->SetName("tlResults1") ... 

Same for the other 2